src/components/CartPage/CartPage.js

```diff
@@ -12,102 +12,178 @@ import './CartPage.scss';
+    const countTotal = userCart.reduce((acc, nextValue) => (acc + this.itemPrice(nextValue.id) * nextValue.chosenQuantity), 0);
+    const totalItemPrice = this.getConvertedPrice(countTotal);
+    const currencyVoucher = this.getConvertedPrice(voucher);
+    const currencyDeliveryCost = this.getConvertedPrice(deliveryCost);
+    const totalAmountToPay = +totalItemPrice + (currencyDeliveryCost - currencyVoucher);
+
+    return (
+      <div className={cx(CN, className)}>
+        <Link className={`${CN}__link`} to="/products">
+          <div className={`${CN}__offer`}>
+            <h2>
           Scoring a good fashion deal is as exhilarating as winning the lottery! Try out your luck!
-          </h2>
-          <div
-            className={`col-6 col-center ${CN}__pseudo-button`}
-          >
+            </h2>
+            <div
+              className={`col-6 col-center ${CN}__pseudo-button`}
+            >
             Continue shopping
+            </div>
           </div>
-        </div>
-      </Link>
-      <YouWillAlsoLove />
+        </Link>
+        <YouWillAlsoLove />
 
-      <div className={`${CN} content`}>
-        <h1 className={`${CN}__title`}>Cart</h1>
-      </div>
-      <div className={`${CN} content`}>
-        {userCart.length ? renderTable()
-          : renderNoItemsInCart()}
-      </div>
-      <div className={`${CN}__checkout content`}>
-        <div className={`${CN}__receipt col-6 col-center`}>
-          <div className="col-10">
-            <p>Subtotal</p>
-            <p>Delivery costs</p>
-            <p>Gift voucher</p>
-            <hr />
-            <p>total price</p>
-          </div>
-          <div className="col-2">
-            <p>{`$${countTotal}`}</p>
-            <p>{`$${deliveryCost}`}</p>
-            <p>{`$${voucher}`}</p>
-            <hr />
-            <p>{`$${countTotal + deliveryCost - voucher}`}</p>
+        <div className={`${CN} content`}>
+          <h1 className={`${CN}__title`}>Cart</h1>
+        </div>
+        <div className={`${CN} content`}>
+          {userCart.length ? this.renderTable()
+            : this.renderNoItemsInCart()}
+        </div>
+        <div className={`${CN}__checkout content`}>
+          <div className={`${CN}__receipt col-6 col-center`}>
+            <div className="col-10">
+              <p>Subtotal</p>
+              <p>Delivery costs</p>
+              <p>Gift voucher</p>
+              <hr />
+              <p>total price</p>
+            </div>
+            <div className="col-2">
+              <p>
+                <>{symbol}</>
+                {`${totalItemPrice}`}
+              </p>
+              <p>
+                <>{symbol}</>
+                {`${currencyDeliveryCost}`}
+              </p>
+              <p>
+                <>{symbol}</>
+                {`${currencyVoucher}`}
+              </p>
+              <hr />
+              <p>
+                <>{symbol}</>
+                {`${totalAmountToPay}`}
+              </p>
```

Comment:
this looks a little bit strange for me

Reply:
this is symbol rendering depending on currency, thats why I've put it in separate {}.
Linter requires ti write it in separate lines.

Reply:
done